src/vm/analysis/arithmetic_checker/tests.rs

```diff
@@ -78,42 +78,62 @@ fn test_bad_defines() {
 }
 
 #[test]
-fn test_variables_fail_arithmetic_check() {
+fn test_bad_defines_clarity2() {
```

Comment:
Not sure if it makes sense to add a "bad defines" test specifically for clarity2, considering we aren't planning on adding any define statements to Clarity2. 

Instead, maybe just wrap the for loop on line 70 (`for (contract, error) in tests.iter() {`), in a for loop that iterates through clarity versions 1 and 2